README.org

```diff
@@ -21,20 +27,45 @@ token directly:
   (setq cody--access-token "sgp_etc")
 #+end_src
 
-To try it out, evaluate the buffer.
+To try it out, evaluate the buffer and `M-x cody-login'.
+
+Note: For now, if you are using the extension and want ripgrep support,
+you will want to add this to your `~/.bashrc` or equivalent:
+
+#+begin_src shell-script
+  export MOCK_RG_PATH="/opt/homebrew/bin/rg" # path to your rg binary
+#+end_src
+
+** Sample Configuration
+
+You can create a Sourcegraph access token at https://sourcegraph.com/users/yourname/settings/tokens.
```

Comment:
This will link to create a token for the logged in user. I think we rewrite `/user/` to `/users/<logged in user>`.

```suggestion
You can create a Sourcegraph access token at https://sourcegraph.com/user/settings/tokens/new-token .
```